### PR TITLE
Switch Documentation

### PR DESCRIPTION
**Pull request by DanielleRameau**

### CODE REVIEW

File: website/src/categories/components/switch/usage.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ eleventyNavigation:
 
 ```html
 
-<sl-switch size="md" checked hint="Switch helper text">switch text</sl-switch>
+<sl-switch size="md" checked>switch text</sl-switch>
```
Comment (jpzwarte) [inline website/src/categories/components/switch/usage.md:18]: We should always start with a capital letter?

**Review by jpzwarte — APPROVED** [review]

File: website/src/categories/components/switch/usage.md [hunk 2]
```diff
@@ -98,32 +68,39 @@ If your setting has multiple states or requires more than a simple on/off toggle
...
+### Switch
 
 |Item|Options|Description|
 |-|-|-|
-|Sizes|`'small', 'medium', 'large'`|...|
-|Status |`...`|...|
-|State|`...`|...|
-|Icon|`...`|...|
+|Sizes|`sm` `md` `lg`|The switch come in three sizes: small, medium (default) and large. |
+|Orientation |`left` `right` `vertical`|The switch has 3 different lay-out possibilities, providing flexibility to align with your design requirements.|
```
Comment (anna-lach) [inline website/src/categories/components/switch/usage.md:78]: We no longer have the `vertical` orientation in the code, maybe it should be updated in Figma as well?

File: website/src/categories/components/switch/usage.md [hunk 3]
```diff
@@ -98,32 +70,39 @@ If your setting has multiple states or requires more than a simple on/off toggle
...
```
[the remark this answers is not in the document]
Comment (DanielleRameau) [reply website/src/categories/components/switch/usage.md:78]: We will add this later in the behavior part when finetuning the pages.

File: website/src/categories/components/switch/usage.md (hunk 2 above)
Comment (DanielleRameau) [reply website/src/categories/components/switch/usage.md:78]: will remove it here in the documentation and make an issue to remove it figma

File: website/src/categories/components/switch/usage.md [hunk 4]
```diff
@@ -98,32 +68,39 @@ If your setting has multiple states or requires more than a simple on/off toggle
...
+### Switch
 
 |Item|Options|Description|
 |-|-|-|
-|Sizes|`'small', 'medium', 'large'`|...|
-|Status |`...`|...|
-|State|`...`|...|
-|Icon|`...`|...|
+|Sizes|`sm` `md` `lg`|The switch come in three sizes: small, medium (default) and large. |
+|Orientation |`left` `right`|The switch has 2 different lay-out possibilities, providing flexibility to align with your design requirements.|
```
Comment (Diaan) [inline website/src/categories/components/switch/usage.md:78]:
```suggestion
|Orientation |`left` `right`|The switch has 2 different lay-out possibilities relating to the positions of the label and switch relative to each other.|
```

**Pull request merged**